client/landing/gutenboarding/onboarding-block/signup-form/index.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import React, { useState } from 'react';
+import { Button, ExternalLink, TextControl, Modal } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+import { __ as NO__, _x as NO_x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { USER_STORE } from '../../stores/user';
+import './style.scss';
+
+const SignupForm = () => {
+	const [ emailVal, setEmailVal ] = useState( '' );
+	const { createAccount } = useDispatch( USER_STORE );
+	const isFetchingNewUser = useSelect( select => select( USER_STORE ).isFetchingNewUser() );
+	const newUser = useSelect( select => select( USER_STORE ).getNewUser() );
+	const newUserError = useSelect( select => select( USER_STORE ).getNewUserError() );
+
+	const handleSignUp = ( event: React.FormEvent< HTMLFormElement > ) => {
+		event.preventDefault();
+
+		createAccount( { email: emailVal, is_passwordless: true, signup_flow_name: 'gutenboarding' } );
+	};
+
+	return (
+		<Modal
+			className="signup-form"
+			isDismissible={ false }
```

Comment:
Should be `isDismissable`

Reply:
[Reading the GB source](https://github.com/WordPress/gutenberg/blob/c079120ce39044e53ea41a6a1a246aeec8195433/packages/components/src/modal/index.js#L113), `dismissable` is deprecated. It also doesn't hide the close button, which is what I was trying to achieve.

Unfortunately the TypeScript types don't line up with this so I'll push an update to those too p1579736961026000-slack-C45SNKV4Z.